lib/modules/search.js

```diff
@@ -0,0 +1,39 @@
+let cheerio = require('cheerio');
+let Service = require('./service');
+
+class Search extends Service {
+  constructor(config) {
+    super(config);
+  }
+
+  query(opt) {
+    let name     = opt.name   || '';
+    let stem     = opt.stem   || '';
+    let member   = opt.member || '';
+    let owner    = opt.owner  || '';
+    let type     = opt.effective ? 'effective' : 'direct';
+    let scope    = opt.scope === 'all' ? 'all' : 'one';
+    let titletoo = opt.titletoo ? '&titletoo=yes' : '';
+
+    return this._get(`search/?name=${name}&stem=${stem}&member=${member}&owner=${owner}&type=${type}&scope=${scope}${titletoo}`)
```

Comment:
Instead, use the `query-string` module to stringify the params like https://github.com/UWFosterIT/node-uwsws/blob/master/src/modules/college.js#L18